.buildkite/x-pack/pipeline.xpack.filebeat.yml

```diff
@@ -217,6 +247,9 @@ steps:
           set -euo pipefail
           source .buildkite/scripts/install_macos_tools.sh
           cd x-pack/filebeat && mage build unitTest
+        retry:
+          automatic:
+           - limit: 3
```

Comment:
Is there a reason we haven't added retries to the two Cloud module tests below?

Reply:
As I mentioned [here](https://github.com/elastic/beats/pull/39174#issuecomment-2075110905) - Cloud tests are not being retried in Jenkins,, same is with k8s tests. Though the [retry policy stated on Jenkins](https://github.com/elastic/beats/blob/50e173aebb6ef064adb3f1a97dcef52b998af55d/Jenkinsfile#L1124-L1128) does not provide any context and I'm not sure if that is actual/valid.

Reply:
> As I mentioned [here](https://github.com/elastic/beats/pull/39174#issuecomment-2075110905) - Cloud tests are not being retried in Jenkins,, same is with k8s tests. Though the [retry policy stated on Jenkins](https://github.com/elastic/beats/blob/50e173aebb6ef064adb3f1a97dcef52b998af55d/Jenkinsfile#L1124-L1128) does not provide any context and I'm not sure if that is actual/valid.

Sorry I missed the earlier comment. Makes sense.